shared/src/main/resources/core-site.xml

```diff
@@ -5,26 +5,8 @@
     <description>Register GCS Hadoop filesystem</description>
   </property>
   <property>
-    <name>fs.gs.auth.service.account.enable</name>
-    <value>false</value>
-    <description>Force OAuth2 flow</description>
-  </property>
-  <property>
-    <name>fs.gs.auth.client.id</name>
-    <value>32555940559.apps.googleusercontent.com</value>
-    <description>Client id of Google-managed project associated with the Cloud SDK</description>
-  </property>
-  <property>
-    <name>fs.gs.auth.client.secret</name>
-    <value>ZmssLNjJy2998hD4CTg2ejr2</value>
-    <description>Client secret of Google-managed project associated with the Cloud SDK</description>
-  </property>
-  <property>
-    <name>fs.gs.project.id</name>
-    <value>_THIS_VALUE_DOES_NOT_MATTER_</value>
-    <description>
-      This value is required by GCS connector, but not used in the tools provided here.
-      The value provided is actually an invalid project id (starts with `_`).
-    </description>
+    <name>fs.gs.auth.type</name>
+    <value>APPLICATION_DEFAULT</value>
+    <description>configures Application Default Credentials authentication</description>
```

Comment:
@shnapz I've tried the user config but this required a refresh token to be provided,
I think this is what we look for, as this will get credentials from the `gcloud` cli

Reply:
According to [docs](https://github.com/GoogleCloudDataproc/hadoop-connectors/blob/master/gcs/CONFIGURATION.md#authentication) you are right!